README.md

```diff
@@ -68,6 +68,15 @@ Running the AKS Periscope requires you to have a storage account associated with
 
 For more information on Diagnostics settings, visit [Create diagnostic settings to send platform logs and metics to different destinations](https://docs.microsoft.com/azure/azure-monitor/platform/diagnostic-settings).
 
+### Configure CI/CD 
+
+Right click on your AKS cluster and click on **Configure CI/CD** to create a CI/CD pipeline targetting your cluster. 
```

Comment:
💡 Just for other folks as thought share, not sure if we want to underline that these 2 commands are the hooks for the extension `Deploy to Azure`? aka https://marketplace.visualstudio.com/items?itemName=ms-vscode-deploy-azure.azure-deploy

Reply:
Expanding CI/CD makes sense.  However we should not need to take a dependency on Deploy to Azure extension for "Browse Pipelines". Why is that needed ? cc: @kanika1894

Reply:
* Indeed let me correctly rephrase that out of 2 command only one I.e. configurePipeline is the one which has dependency and seems like browsePipeline is just url forwarding. 

* rectified that in my comment above, thank you 🙏

Reply:
awesome. Have updated the readme for configure.